kimchi/src/error.rs

```diff
@@ -41,6 +41,9 @@ pub enum VerifyError {
     #[error("the public input is of an unexpected size (expected {0})")]
     IncorrectPubicInputLength(usize),
 
+    #[error("the previous challenges have an unexpected length (expected {0})")]
+    IncorrectPrevChallengesLength(usize),
```

Comment:
might want to add the length of the one received as well, like in https://github.com/o1-labs/proof-systems/pull/609/files#diff-fc95db4d289fd33bc7fbefe8a487926ce8b47cbe71f047a1dd6933d7444b5d5aR150